packages/manager/apps/web/client/app/domain/zone/record/domain-zone-record.controller.js

```diff
@@ -8,6 +8,8 @@ import isEmpty from 'lodash/isEmpty';
 import isString from 'lodash/isString';
 import trim from 'lodash/trim';
 import set from 'lodash/set';
+import { Environment } from '@ovh-ux/manager-config';
+import { CANADA_REGION } from '../../../../../../../modules/config/src/environment/environment.constants';
```

Comment:
Why do you need to import `environment.constants` like this ?
If you really need it, constants should be exported by `@ovh-ux/manager-config` ?

Here `import ... from '@ovh-ux/manager-config';` and `import ... from '../../../../../../../modules/config/src/` target the same package.

Reply:
I had try create a static method `isCanadaRegion()`.
When i try to call it like so `Environment.isCanadaRegion()` i got error.
Now i understand the mistake the `Environment` is already an instance object not a Class.

So i don't need anymore the `CANADA_REGION` import.